worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitPlayer.java

```diff
@@ -66,6 +68,7 @@ public BukkitPlayer(Player player) {
     public BukkitPlayer(WorldEditPlugin plugin, Player player) {
         this.plugin = plugin;
         this.player = player;
+        this.audience = BukkitAudiences.create(plugin).player(player);
```

Comment:
You should have one instance of `BukkitAudiences.create(plugin)` cached and re-use the created factory to create your audiences.